### ISSUE TITLE
implementation of Proximal Bregman Response Function w.r.t the linear layer of the BERT model

### ISSUE THREAD

**Issue by huutiendang** [nrimsky/InfluenceFunctions#1]
Thank you for your implementation. I encountered difficulties while attempting to apply PBRF to the final layer (linear) when fine-tuning a BERT model for text classification. Would it be possible for you to share the code for PBRF in this specific context? @nrimsky 

Thank you again!

**Comment by nrimsky** [comment]
Good idea - I will take a look at this at some point, am currently caught up with other projects but want to get back to Influence Functions stuff shortly

**Comment by huutiendang** [comment]
> Good idea - I will take a look at this at some point, am currently caught up with other projects but want to get back to Influence Functions stuff shortly

Thank you Nina @nrimsky , I hope to hear from you soon!
